buildpack.toml

```diff
@@ -155,22 +155,44 @@ api = "0.7"
   [[metadata.dependencies]]
     cpes = ["cpe:2.3:a:oracle:jdk:20.0.2:*:*:*:*:*:*:*"]
     id = "jdk"
-    name = "oracle-jdk"
+    name = "Oracle JDK"
     purl = "pkg:generic/oracle-jdk@20.0.2?arch=amd64"
     sha256 = "499b59be8e3613c223e76f101598d7c28dc04b8e154d860edf2ed05980c67526"
-    stacks = ["io.buildpacks.stacks.bionic", "io.paketo.stacks.tiny", "*"]
+    stacks = ["*"]
     uri = "https://download.oracle.com/java/20/archive/jdk-20.0.2_linux-x64_bin.tar.gz"
     version = "20.0.2"
 
     [[metadata.dependencies.licenses]]
       type = "Oracle No-Fee Terms and Conditions License"
       uri = "https://www.oracle.com/downloads/licenses/no-fee-license.html"
 
-[[stacks]]
-  id = "io.paketo.stacks.tiny"
+  [[metadata.dependencies]]
+    cpes = ["cpe:2.3:a:oracle:jdk:17.0.8:*:*:*:*:*:*:*:*"]
+    id = "native-image-svm"
+    name = "Oracle GraalVM"
+    purl = "pkg:generic/graalvm-jdk@17.0.8"
+    sha256 = "2d6696aa209daa098c51fefc51906aa7bf0dbe28dcc560ef738328352564181b"
+    stacks = ["*"]
+    uri = "https://download.oracle.com/graalvm/17/archive/graalvm-jdk-17.0.8_linux-x64_bin.tar.gz"
+    version = "17.0.8"
 
-[[stacks]]
-  id = "io.buildpacks.stacks.bionic"
+    [[metadata.dependencies.licenses]]
+      type = "Oracle No-Fee Terms and Conditions License"
```

Comment:
Oracle GraalVM is available under the [GraalVM Free Terms and Conditions license](https://www.oracle.com/downloads/licenses/graal-free-license.html).

Reply:
Thanks! I was going to ask and you beat me to it. Updated.